### DISCUSSION TITLE
Dikey telefon ekranı, yatay telefon ekranı için ad belirleme

### DISCUSSION THREAD

**Discussion by volkantash** [turkce-yazilim-konati/yazilimca#84] [Soru-Yanıt]
Selamlar.

Bilirsiniz. Cep telefonlarını yatay tutuğumuzda ekranı geniş yanından kullanır duruma ereriz. Telefon ekranlarının yatay ve dikey tutularak kullanılabilir olması bir yana bu ekranlar için adlar gerekmektedir. Telefon ekranı dediğimizde ilk başta dikey tutuş göz önüne gelse de yatay tutuş için bir ad koymalıyız.

İngilizcede PORTRAIT ile LANDSCAPE sözcükleri karşımıza çıkıyor.

![640px-800x518_smartphone_portrait_vs_landscape_orientation](https://user-images.githubusercontent.com/25375593/191626442-4d159d6d-d038-443b-a993-b209644f7ed2.png)

Siz ne ad koyardınız?

**Comment by tolga-celik** [comment]
Dikgör
Yangör

Dikdur
Yandur

Dikserp
Yanserp

**Reply by volkantash** [reply]
Öneriler için sağ olun. 🤗

Yabancı terimlere Türkçe karşılık arayışları için kılavuz niteliğinde olan şu içeriği izlemeyi unutmayın. https://www.youtube.com/watch?v=0QiuqwPv25A
